ruby-3.1.advisories.yaml

```diff
@@ -3,97 +3,97 @@ package:
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2022-1343:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2022-1434:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2022-1473:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2022-2068:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2022-2097:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2022-3358:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2022-3602:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2022-3786:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2022-3996:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2022-4203:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2022-4304:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2022-4450:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2023-0215:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2023-0216:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2023-0217:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2023-0286:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
       status: not_affected
       justification: component_not_present
 
   CVE-2023-0401:
-    - timestamp: 2023-06-23T03:04:27.00Z
+    - timestamp: 2023-06-23T03:04:27Z
```

Comment:
Why did all of these update?

Reply:
I think from an decode/encode cycle. This is something I'd like to standardize soon, perhaps with the advisory schema updates.

Reply:
Going to resolve for now to keep this moving, but ping me if you have more thoughts/suggestions!